src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      return (this.result += value);
```

Comment:
[CODE STYLE] - Instead of returning the result of the arithmetic operation, you should perform the operation and then return the calculator object to maintain method chaining. The current implementation breaks the chaining feature because it returns the new value of the result instead of the calculator object itself.